src/datadog/runtime_id.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+// This component provides a `class`, `RuntimeID`, that is a wrapper around an
+// RFC 4122 UUIDv4. `RuntimeID` identifies the current run of the application in
+// which this tracing library is embedded.
+//
+// See `TracerConfig::runtime_id`, declared in `tracer_config.h`.
+
+#include <string>
+
+namespace datadog {
+namespace tracing {
+
+class RuntimeID {
```

Comment:
From my understanding, `RuntimeID` is a `std::string` in a tuxedo. How about `using RuntimeID = std::string` and `generate()` as free function? We lose strong typing in the process but we don't use it for now.

Reply:
That's similar to what Caleb had before.

The reason I dressed our string in formal wear is because we were exposing the concept of RuntimeID to the user for the first time. This strict technique guarantees that RuntimeID is a UUID. Not that anybody was likely to set it to anything else...